packages/api/src/hooks/usePOI.ts

```diff
@@ -47,7 +47,7 @@ const usePOI = () => {
         const user_country_code = get_settings_data?.citizen || get_settings_data?.country_code;
         const matching_residence_data = residence_list_data?.find(r => r.value === user_country_code);
         const is_idv_supported = matching_residence_data?.identity?.services?.idv?.is_country_supported;
-        const is_onfido_supported = matching_residence_data?.identity?.services?.onfido?.documents_supported;
```

Comment:
`documents_supported` is an empty object `{}`, `is_onfido_supported` will always be truthy